src/main/scala/com/ubirch/discovery/services/consumer/DiscoveryService.scala

```diff
@@ -278,7 +279,7 @@ abstract class AbstractDiscoveryService(storer: Storer, config: Config, lifecycl
       import scala.concurrent.duration._
       // will fail if the graph is empty
       val healthReportJG = try {
-        Await.result(gc.g.V().limit(1).promise(), 50.millis) match {
+        Await.result(gc.g.V().has(Key[String]("hash"), jgHealthCheckHashValue).limit(1).promise(), 50.millis) match {
```

Comment:
I am wondering why we block here?

Reply:
It doesn't included in the scope of this ticket but fixing it would be an another tech dept candidate

Reply:
Perfect. Please create a tech debt ticket so that we don't forget that.